.run/PostHog.run.xml

```diff
@@ -14,6 +14,7 @@
       <env name="PRINT_SQL" value="1" />
       <env name="PYTHONUNBUFFERED" value="1" />
       <env name="SKIP_SERVICE_VERSION_REQUIREMENTS" value="1" />
+      <env name="CAPTURE_TIME_TO_SEE_DATA" value="0" />
```

Comment:
👀

Reply:
Whenever I turn it on I have to go grepping in the code to remind myself what the env var is. So, add it in. But turned off. I suppose that line of reasoning would have every boolean var in the run file... but I turn this one on a lot :)